python/cudf/cudf/core/index.py

```diff
@@ -748,7 +776,6 @@ def get_slice_bound(self, label, side, kind):
         raise (NotImplementedError)
 
     def __array_function__(self, func, types, args, kwargs):
```

Comment:
There's a bunch of `return NotImplemented` in here, is this intended? Normally it should be `raise NotImplementedError`